cmd/kubeadm/app/cmd/init.go

```diff
@@ -248,6 +251,14 @@ func AddInitOtherFlags(
 		uploadCerts, options.UploadCerts, *uploadCerts,
 		"Upload certfificates to kubeadm-certs secret.",
 	)
+	flagSet.StringVar(
+		certificateKey, options.CertificateKey, "",
+		"Use this key to encrypt the certificate secrets.",
+	)
+	flagSet.BoolVar(
+		skipCertificateKeyPrint, options.SkipCertificateKeyPrint, *skipCertificateKeyPrint,
+		"Skip printing of the certificate key used to encrypt certs by 'kubeadm init'.",
```

Comment:
`Skip printing of the key used to encrypt the control-plane certificates.`

Reply:
Or even better `Don't print the key used to encrypt the control-plane certificates.`